# Not final, Will be edited.
string agent_id
string command_type
float32[] parameters